srcs/juloo.keyboard2/Config.java

```diff
@@ -60,6 +63,9 @@ private Config(Context context, IKeyEventHandler h)
     keyVerticalInterval = res.getDimension(R.dimen.key_vertical_interval);
     keyHorizontalInterval = res.getDimension(R.dimen.key_horizontal_interval);
     preciseRepeat = true;
+    lockShift = true;
+    lockCtrl = true;
+    lockAlt = true;
```

Comment:
This is the default value when the option is unset, should be `false` (except for shift).